main.go

```diff
@@ -30,7 +32,7 @@ func main() {
 	r.HandleFunc("/library/{id}/checkout", checkoutBook).Methods("PUT")
 	r.HandleFunc("/library/{id}/checkin", checkinBook).Methods("PUT")
 
-	http.ListenAndServe(cfg.BindAddr, r)
+	http.ListenAndServe(":"+cfg.BindAddr, r)
```

Comment:
I would suggest removing the colon ":" from here, and keeping it as part of BindAddr (for consistency with other services)